File: aula3/cpp/desafio3.cpp
#include <iostream>
using namespace std;

int main() {
    float soma = 0, nota;

    for (int i = 0; i < 5; i++) {
        cout << "Digite a nota: ";
        cin >> nota;
        soma += nota;
    }

    cout << "Média: " << soma / 5 << endl;
    return 0;
}

File: aula3/java/Desafio3.java
import java.util.Scanner;

public class Desafio3 {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        double soma = 0;

        for (int i = 0; i < 5; i++) {
            System.out.print("Digite a nota: ");
            soma += scanner.nextDouble();
        }

        System.out.println("Média: " + soma / 5);
        scanner.close();
    }
}
